regolith/helpers/l_contactshelper.py

```diff
@@ -0,0 +1,131 @@
+"""Helper for finding and listing contacts from the contacts.yml database.
+Prints name, institution, and email (if applicable) of the contact.
+"""
+import datetime 
+import dateutil
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    fuzzy_retrieval,
+)
+
+TARGET_COLL = "contacts"
+HELPER_TARGET = "l_contacts"
+
+def subparser(subpi):
+    subpi.add_argument("-n", "--name", help='list of names or name fragments (multiple arguments allowed) to use to find contacts', nargs = '+')
+    subpi.add_argument("-i", "--inst", help='institution or an institution fragment (single argument only) to use to find contacts')
+    subpi.add_argument("-d", "--date", help='approximate date in ISO format (YYYY-MM-DD) corresponding to when the contact was entered in the database. Comes with a default range of 4 months centered around the date; change range using --range argument')
+    subpi.add_argument("-r", "--range", help='range (in months) centered around date d specified by --date, i.e. (d +/- r/2)')
+    subpi.add_argument("-o", "--notes", help='use list of notes or miscellaneous information to use (multiple arguments allowed) to find contacts', nargs = '+')
```

Comment:
how about for the help "fragment of text that will be searched in the notes field of the contact.  If fragments are specified in name and here, AND logic will apply "

Reply:
Done